### DISCUSSION TITLE
Media folder

### DISCUSSION THREAD

**Discussion by mojubouri** [spatie/laravel-medialibrary#2857] [Q&A]
Hello Dear,

I have a problem with the media folder in public, any files I upload it the folder will be 700 permission, and I tried to change the folder -R chmod but it's still the same problem that's happened on this version only  

![Screen Shot 2022-03-22 at 4 47 07 PM](https://user-images.githubusercontent.com/16868721/159496199-8e935e88-81f6-4381-a364-bb23d4ebcc5f.png)

**Comment by PaolaRuby** [comment]
Duplicate of #2858

**Reply by mojubouri** [reply]
There is any answers for this
